include/mbedtls/ssl.h

```diff
@@ -1758,8 +1758,8 @@ void mbedtls_ssl_conf_read_timeout( mbedtls_ssl_config *conf, uint32_t timeout )
 
 #if defined(MBEDTLS_SSL_RECORD_CHECKING)
 /**
- * \brief          Check whether a buffer contains a valid, fresh
- *                 and authentic record (DTLS only).
+ * \brief          Check whether a buffer contains a valid and authentic record
+ *                 and verify it has not been seen before (DTLS only).
```

Comment:
Nitpick: `... and authentic record that has not been seen before.`

Reply:
Done